app/helpers/keywords_helper.rb

```diff
@@ -0,0 +1,9 @@
+module KeywordsHelper
+
+  def keyword_bodies(abstract)
```

Comment:
This is fine as a helper, but I would probably put this as a method on the Abstract model. I think you'd find others would disagree, but to my mind it makes the code cleaner. 

Even better would be to build a `has_keyword?` method on Abstract to tidy up line 8 of `abstracts/show.html.erb`  - this would encompass all the logic in one place.